sqlglot/dialects/postgres.py

```diff
@@ -277,6 +278,7 @@ class Generator(generator.Generator):
             exp.DataType.Type.BINARY: "BYTEA",
             exp.DataType.Type.VARBINARY: "BYTEA",
             exp.DataType.Type.DATETIME: "TIMESTAMP",
+            exp.DataType.Type.INET: "INET",
```

Comment:
Ditto for the base generator.

Reply:
Thanks George, I've taken the Postgres specific code out which included moving the unit test, I'm seeing something I wasn't expecting in the tests. Something I didn't see before updating the Postgres dialect code.

for the input `INET '127.0.0.1'` it translates to `CAST('127.0.0.1' AS INET)` as expected, but for the input `CAST('127.0.0.1' AS INET)` it translates to `TRY_CAST('127.0.0.1' AS INET)`. Is this expected behaviour from an implementation perspective? 

This appears to mean if I ran the output of a transpile through the engine again, I'd get a different response back.

Reply:
Yes, it's expected: depending on the source dialect, cast is parsed into either exp.Cast or exp.TryCast. There is a flag in the parser that determines whether the cast is to be parsed as strict or not.

Reply:
Also, forgot to mention: some dialects (e.g. postgres) override the SQL generation logic for TryCast so that it actually transpiles back to `CAST` for them (even though the AST node is TryCast).